File: Lab.PresentationLayer/src/app/shared/models/requisition.dto.ts
import { DtoBase } from "./dto-base.dto";
import { TestDto } from "./test.dto";

export class RequisitionDto extends DtoBase<number>{
        requisitionId:string = "";
        timeSampleTaken?:Date;
        firstName = "";
        surname ="";
        gender : "M"|"F"|"U" = "U";
        dateOfBirth?:Date;
        age = 0
        mobileNumber = "";
        requestedTestIdentifiers: Array<number> = [];
        requestedTests : Array<TestDto> = [];
}

File: Lab.BusinessLogicLayer/Models/DataTransferObjects/RequisitionDto.cs
using Lab.BusinessLogicLayer.Models.DataTransferObjects.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab.BusinessLogicLayer.Models.DataTransferObjects
{
    public class RequisitionDto : DtoBase<int>
    {
        public string RequisitionId => $"{base.Id:D4}";
        public DateTime? TimeSampleTaken { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string Surname { get; set; } = string.Empty;
        public string Gender { get; set; } = string.Empty;
        public DateTime? DateOfBirth { get; set; }
        public int Age { get;set;}
        public string MobileNumber { get; set; } = string.Empty;
        public IEnumerable<int> RequestedTestIdentifiers { get; set; } = Enumerable.Empty<int>();
        public List<TestDto> RequestedTests { get; set; } = new();
        public string RequestedTestId = string.Empty;
        public string RequestedTestsDescription { get; set; } = string.Empty;
    }
}
